app/forms/thredded/topic_form.rb

```diff
@@ -22,12 +22,8 @@ def self.model_name
       Thredded::Topic.model_name
     end
 
-    def categories
-      topic.messageboard.categories
-    end
-
     def category_options
-      categories.map { |cat| [cat.name, cat.id] }
+      messageboard.categories.map { |cat| [cat.name, cat.id] }
```

Comment:
FeatureEnvy: Thredded::TopicForm#category_options refers to 'cat' more than self (maybe move it to another class?)